person.rb

```diff
@@ -1,27 +1,31 @@
 require './nameable.rb'
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Setters for @rentals should be present since `@rantals` is accessed in the Rental class.

Rental class line 11:
```rb
book.rentals << self
```